purchase_cancel_qty/views/purchase_order.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="purchase_order_form_view">
+        <field name="name">purchase.order.form (in purchase_cancel_qty)</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+
+            <xpath expr="//field[@name='order_line']/tree/field[@name='product_qty']" position="after">
+                <field name="initially_ordered_qty" invisible="not context.get('show_purchase', False)"/>
+                <field name="cancelled_qty" invisible="not context.get('show_purchase', False)"/>
```

Comment:
would it make sense to hide `product_qty` when `show_purchase` is True?

Reply:
If we do that, user will have to calculate himself what quantity is actually processed.